pkg/scheduler/api/pod_info.go

```diff
@@ -117,14 +118,27 @@ func GetPodRevocableZone(pod *v1.Pod) string {
 	return ""
 }
 
-// GetPodTopologyPolicy return volcano.sh/numa-topology-policy value for pod
-func GetPodTopologyPolicy(pod *v1.Pod) string {
+// GetPodTopologyInfo return volcano.sh/numa-topology-policy value for pod
```

Comment:
What's the meaning of volcano.sh/numa-topology-policy value? Does it contains the policy on pod and the allocated numa resource?

Reply:
yep, you're right